src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/model/HdcLicence.kt

```diff
@@ -15,5 +17,5 @@ data class HdcLicence(
   val firstNightCurfewHours: FirstNight? = null,
 
   @Schema(description = "The curfew times for the person following the first night")
-  val curfewHours: CurfewHours? = null,
+  val curfewTimes: List<CurfewTimes?> = emptyList(),
```

Comment:
**suggestion:** This is saying that the list can contain nulls which I don't think is possible or probably desirable. I think it's probably best to remove all nullability here